appveyor.yml

```diff
@@ -52,6 +52,7 @@ matrix:
     - OTP_VERSION: 21.2
 
 init:
+  - ps: Update-AppveyorBuild -Version "$env:appveyor_build_version-$($env:appveyor_repo_commit.substring(0,7))"
```

Comment:
I see syntax for lookup of variable from environment [documented](https://www.appveyor.com/docs/build-configuration/#setting-environment-variables-in-build-script):
> Once a PowerShell variable is set you access it by `$env:MY_VARIABLE`.